samples/client/petstore/python-experimental/tests/test_api_client.py

```diff
@@ -1,178 +0,0 @@
-# coding: utf-8
```

Comment:
Please revert all files in python-experimental/tests back to what the master branch has, we need these

Reply:
Can you resolve this conversation?